extensions/amp-analytics/0.1/crypto-impl.js

```diff
@@ -20,29 +20,62 @@ import {getService} from '../../../src/service';
 export class Crypto {
 
   constructor(win) {
-    this.win = win;
+    if (win.crypto) {
+      this.subtle = win.crypto.subtle || win.crypto.webkitSubtle;
```

Comment:
How do I check that, are externs listed anywhere?
Didn't see [here](http://closureplease.com/externs/)